src/rwe/GameScene.cpp

```diff
@@ -147,30 +147,31 @@ namespace rwe
 
     float computeSoundCeiling(int soundCount)
     {
+        float c = static_cast<float>(soundCount);
         assert(soundCount > 0);
         if (soundCount <= 4)
         {
-            return soundCount;
+            return c;
         }
 
         if (soundCount <= 8)
         {
-            return 4 + ((soundCount - 4) * 0.5f);
+            return 4.f + ((c - 4.f) * 0.5f);
```

Comment:
non-blocking style nit: I prefer float literals to always include on decimal place e.g. `4.0f`